From f6805126b9300502fa3e54bfe7c1da7fc0350a74 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Torsten=20Sch=C3=B6nfeld?= <kaffeetisch@gmx.de>
Date: Sun, 27 Nov 2011 12:24:30 +0100
Subject: [PATCH] Fix compilation against cairo < 1.6.0

Protect CAIRO_FONT_TYPE_QUARTZ and CAIRO_FONT_TYPE_USER with version guards.

https://rt.cpan.org/Ticket/Display.html?id=72576
---
 CairoFont.xs |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/CairoFont.xs b/CairoFont.xs
index 41a4331..5ecd5ac 100644
--- a/CairoFont.xs
+++ b/CairoFont.xs
@@ -28,8 +28,12 @@ get_package (cairo_font_face_t *face)
 
 	    /* These aren't wrapped yet: */
 	    case CAIRO_FONT_TYPE_WIN32:
+#if CAIRO_VERSION >= CAIRO_VERSION_ENCODE(1, 6, 0)
 	    case CAIRO_FONT_TYPE_QUARTZ:
+#endif
+#if CAIRO_VERSION >= CAIRO_VERSION_ENCODE(1, 8, 0)
 	    case CAIRO_FONT_TYPE_USER:
+#endif
 		package = "Cairo::FontFace";
 		break;
 
-- 
1.7.5.4

